appveyor.yml

```diff
@@ -32,44 +32,53 @@
     - '%AppData%\npm-cache'
 
   environment:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
```

Comment:
That is definitely an option (and likewise in `nvm` and hence `node-chakracore`), but at this time we just preferred using `nvs` since it added [support for CI](https://github.com/jasongin/nvs/blob/master/doc/CI.md) recently.